### DISCUSSION TITLE
Release of v0.0.6

### DISCUSSION THREAD

**Discussion by northern-64bit** [scikit-multilearn-ng/scikit-multilearn-ng#14] [Announcements]
## What's Changed
* Fix bug in ClassifierChain by @northern-64bit in https://github.com/scikit-multilearn-ng/scikit-multilearn-ng/pull/9
* Enables shuffling in iterative_train_test_split by @northern-64bit in https://github.com/scikit-multilearn-ng/scikit-multilearn-ng/pull/8
* Adds classification with heterogeneous features by @northern-64bit in https://github.com/scikit-multilearn-ng/scikit-multilearn-ng/pull/10
* Adds InstanceBasedLogisticRegression and StructuredGridSearchCV by @northern-64bit in https://github.com/scikit-multilearn-ng/scikit-multilearn-ng/pull/11
* Smile algorithm by @northern-64bit in https://github.com/scikit-multilearn-ng/scikit-multilearn-ng/pull/13

**Full Changelog**: https://github.com/scikit-multilearn-ng/scikit-multilearn-ng/compare/v0.0.5...v0.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/scikit-multilearn-ng/scikit-multilearn-ng/releases/tag/v0.0.6'>Release of v0.0.6</a>.</em>
